tools/integration-tests/assets/entrypoint.sh

```diff
@@ -5,5 +5,12 @@ cp -rp /tmp/assets/* /assets
 chmod 777 /assets/*
 echo "assets:"
 ls /assets
-echo "running tests..."
-go test ./tests/"${TEST_NAME}" -v -timeout 30m -count=1
+if [ "x$DEBUG" = "x1" ]; then
```

Comment:
Just curious why are you using `x`?
I saw that `"$DEBUG" = "1"` works perfectly well!
I guess you wanted to make sure it is a string?

I don't care much for the extra character
I just must know you reason to have peace of mind :-P

Reply:
It is a bit common practice in GNU tools. Using "$DEBUG" also works.